server/sonar-db-dao/src/test/java/org/sonar/db/measure/ProjectMeasuresIndexerIteratorTest.java

```diff
@@ -120,6 +121,7 @@ public void does_not_fail_when_quality_gate_has_no_value() throws Exception {
     assertThat(docsById.get("project").getMeasures().getNumericMeasures()).isEmpty();
   }
 
+  @Ignore
   @Test
   public void return_language_distribution_measure() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)